### PR TITLE
User guide renames

### PR DESCRIPTION
**Pull request by dblevins**
There are two commits in this PR:

1. remove any false jakarta packages
2. add any missed jakarta renames

Intent of the division is to make it easier to review since each commit can be browsed separately.

### CODE REVIEW

File: user_guides/jakartaee/src/main/jbake/content/rules-wp.adoc [hunk 1]
```diff
@@ -613,7 +613,7 @@ required for the Jakarta EE 9 Web Profile:
 * jakarta.servlet.jsp.jstl.tlv
 * jakarta.servlet.jsp.tagext
 * jakarta.transaction
-* jakarta.transaction.xa
```
Comment (scottmarlow) [inline user_guides/jakartaee/src/main/jbake/content/rules-wp.adoc:616]: :+1:

File: user_guides/jakartaee/src/main/jbake/content/rules.adoc [hunk 2]
```diff
@@ -686,6 +686,6 @@ required for Jakarta EE 9:
 * jakarta.ws.rs.core
 * jakarta.ws.rs.ext
 * jakarta.ws.rs.sse
-* jakarta.xml.registry
-* jakarta.xml.registry.infomodel
+* javax.xml.registry
```
Comment (scottmarlow) [inline user_guides/jakartaee/src/main/jbake/content/rules.adoc:689]: We should remove the (pruned for EE 9) javax.xml.registry class references.

**Review by alwin-joseph — APPROVED** [review]

File: user_guides/jakartaee/src/main/jbake/content/rules.adoc (hunk 2 above)
Comment (alwin-joseph) [reply user_guides/jakartaee/src/main/jbake/content/rules.adoc:689]: I believe this PR can be merged after this is resolved to remove javax.xml.registry & javax.xml.registry.infomodel

**Review by alwin-joseph — CHANGES_REQUESTED** [review]

**Review by alwin-joseph — APPROVED** [review]
Will remove the pruned techs from user_guides/jakartaee/src/main/jbake/content/rules.adoc  in a separate PR

**Pull request merged**